app/Http/Controllers/Api/ReviewsController.php

```diff
@@ -76,13 +77,12 @@ public function reviews(Request $request)
                 array_push($photos, $review);
             } else {
                 $review['rogue_reportback_item_id'] = $review['rogue_event_id'];
-                array_push($reportbackItems, array_only($review, ['rogue_reportback_item_id', 'status']));
+                array_push($reportbackItems, array_except($review, ['rogue_event_id']));
```

Comment:
@katiecrane after our conversation, I guess it won't be since we won't ever be reviewing in Phoenix and sending to Rogue! Should I delete all the reportback item logic?